es-design-system/pages/molecules/es-collapse.vue

```diff
@@ -8,37 +8,118 @@
                 bootstrap-vue collapse
             </b-link>
         </p>
-        <form>
-            <label for="suggestedVisibleInput">
-                <input
-                    id="suggestedVisibleInput"
-                    v-model="suggestedVisible"
-                    type="checkbox">
-                Toggle collapse programmatically (will be honored until a manual expand or collapse)
-            </label>
-        </form>
-        <EsCollapse
-            id="testId"
-            :visible="suggestedVisible"
-            class="p-450 my-450"
-            @shown="shownEvent"
-            @toggled="toggledEvent">
-            <template #title>
-                <h2 class="mb-0">
-                    My Title
-                </h2>
-            </template>
-            <!-- eslint-disable max-len -->
+        <div class="mt-450 mb-450">
+            <h2>
+                Default EsCollapse
+            </h2>
+            <p>
+                A normal EsCollapse component. Click it to toggle showing it's contents!
+            </p>
+            <EsCollapse
+                id="defaultCollapse"
+                class="ml-450 mr-450">
```

Comment:
If we add `mt-450` here, it'd better match the spacing of the other examples between description and collapse.